libs/src/api/account.js

```diff
@@ -132,6 +132,10 @@ class Account extends Base {
           const ownerWallet = await this.hedgehog.signUp(email, password)
           await this.web3Manager.setOwnerWallet(ownerWallet)
           await this.generateRecoveryLink({ handle: metadata.handle, host })
+          if (this.solanaWeb3Manager) {
+            // Create a user bank if the solana web3 manager is present
+            await this.solanaWeb3Manager.createUserBank()
```

Comment:
Ideally, we would check if a user bank already exists -- or even better we create all the user bank accounts for existing users and then create a user bank account on audius account  creation.

Reply:
yeah, that will be done as an offline migration and then this would just run every time on sign up

Reply:
we need to ship this bit first though so we can do the retroactive migration cleanly offline (and have a stopping point)